api/v1alpha1/gateway_types.go

```diff
@@ -56,9 +56,14 @@ type GatewaySpec struct {
 	// Listeners associated with this Gateway. Listeners define what addresses,
 	// ports, protocols are bound on this Gateway.
 	Listeners []Listener `json:"listeners"`
-	// Routes associated with this Gateway. Routes define
-	// protocol-specific routing to backends (e.g. Services).
-	Routes []core.TypedLocalObjectReference `json:"routes"`
+	// Routes defines routes to associate with the Gateway.
+	//
+	// If unspecified, all routes will be associated to the Gateway.
```

Comment:
I'm not sure how to interpret this. If there are no routes specified here, what routes are being associated?

Reply:
@jpeach I think it comes down to defining the default behavior for route association, closed or open. I created https://github.com/kubernetes-sigs/service-apis/issues/84 to address this issue. In the meantime, I'll update the language here to be default closed.

Reply:
Thanks, I commented there.